### PR TITLE
new models 

### PR DESCRIPTION
**Pull request by elya-hp**

### CODE REVIEW

**Comment by pyinto** [comment]
@elya-hp please register all those models in django admins

File: tms_django/apps/tms/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from apps.tms.tests.factories import (
+    BookedLoadFactory,
+    BrokerFactory,
+    DispatcherProfileFactory,
+    DriverProfileFactory,
+    TruckFactory,
+)
+from django.test import TestCase
+
+
```
Comment (pyinto) [inline tms_django/apps/tms/tests/test_models.py:42]: @elya-hp please add tests for custom id fields where you applied generators, such as `load_id` `generate_load_id` and similar

File: tms_django/apps/tms/models.py [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+
+
+class DriverProfile(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, related_name="driver_profile", on_delete=models.CASCADE)
+    unit_id = models.CharField(max_length=6, unique=True, db_index=True, default=generate_unit_id)
+    phone_number = models.CharField(max_length=255, blank=False)
+    driver_licence = models.CharField(max_length=255, blank=False)
+    date_of_birth = models.DateField(null=True, blank=False)
+    address = models.CharField(max_length=255)
+    truck = models.ForeignKey(Truck, null=True, on_delete=models.CASCADE)
```
Comment (pyinto) [inline tms_django/apps/tms/models.py:31]: @elya-hp Does it mean that a driver may have a link to only one truck? Is linking with 2 or more trucks not viable?

File: tms_django/apps/tms/models.py [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
+    pickup_location = models.CharField(max_length=30)
+    delivery_location = models.CharField(max_length=30)
+    distance = models.IntegerField(null=True, default=None, validators=[MinValueValidator(0)])
+    pallets_quantity = models.IntegerField(null=True, default=None, validators=[MinValueValidator(0)])
+    pallets_weight = models.IntegerField(null=True, default=None, validators=[MinValueValidator(0)])
+    dispatcher = models.ForeignKey("tms.DispatcherProfile", on_delete=models.CASCADE)
+    broker = models.ForeignKey(Broker, on_delete=models.CASCADE)
+    total_rate = models.IntegerField(default=None)
+    driver_rate = models.IntegerField(default=None)
+    invoice_id = models.CharField(max_length=8, unique=True, db_index=True, default=None, null=True)
```
Comment (pyinto) [inline tms_django/apps/tms/models.py]: Why no default value set here? When it should be filled in?

File: tms_django/apps/tms/tests/test_models.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    def test_factory(self):
+        instance = TruckFactory()
+
+        instance.refresh_from_db()
+        self.assertIsInstance(instance, self.model_cls)
+        self.assertIsNotNone(instance.id)
+
+
+# TODO: update tests below (as TestTruckModel) so that they are structured in the same way:
```
Comment (pyinto) [inline tms_django/apps/tms/tests/test_models.py]: this too

File: tms_django/apps/tms/tests/test_models.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+
+class TestDriverModel(TestCase):
+    model_cls = DriverProfile
+
+    def test_factory(self):
+        instance = DriverProfileFactory()
+
+        instance.refresh_from_db()
+        self.assertTrue(instance, self.model_cls)
```
Comment (pyinto) [inline tms_django/apps/tms/tests/test_models.py]: incorrect assert

File: tms_django/apps/tms/tests/test_models.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+
+class TestDispatcherProfileModel(TestCase):
+    model_cls = DispatcherProfile
+
+    def test_factory(self):
+        instance = DispatcherProfileFactory()
+
+        instance.refresh_from_db()
+        self.assertTrue(isinstance(instance, self.model_cls))
```
Comment (pyinto) [inline tms_django/apps/tms/tests/test_models.py]: check what kind of assert is used in TestTruckModel

File: tms_django/apps/users/tests/test_models.py
Comment (elya-hp) [inline tms_django/apps/users/tests/test_models.py:1]: Done

File: tms_django/apps/users/tests/factories.py
Comment (elya-hp) [inline tms_django/apps/users/tests/factories.py:1]: Done

File: tms_django/apps/tms/tests/test_models.py (hunk 6 above)
Comment (elya-hp) [reply tms_django/apps/tms/tests/test_models.py]: I don't understand your request. Please explain it to me

File: tms_django/apps/tms/tests/test_models.py (hunk 5 above)
Comment (elya-hp) [reply tms_django/apps/tms/tests/test_models.py]: Done

File: tms_django/apps/tms/tests/test_models.py (hunk 1 above)
Comment (elya-hp) [reply tms_django/apps/tms/tests/test_models.py:42]: done

File: tms_django/apps/tms/models.py (hunk 2 above)
Comment (elya-hp) [reply tms_django/apps/tms/models.py:31]: Correct. He can be assigned only 1 truck

File: tms_django/apps/tms/tests/test_models.py (hunk 6 above)
Comment (elya-hp) [reply tms_django/apps/tms/tests/test_models.py]: @pyinto I cannot. I added 

```
class TestTruckModel(TestCase):
    model_cls = Truck

    def test_factory(self):
        instance = TruckFactory()

        instance.refresh_from_db()
        self.assertIsInstance(isinstance(instance, self.model_cls))
        self.assertIsNotNone(instance.id)
```

and I received this error. 

self.assertIsInstance(isinstance(instance, self.model_cls))
TypeError: TestCase.assertIsInstance() missing 1 required positional argument: 'cls'

GPT says to take off isinstance

Comment (pyinto) [reply tms_django/apps/tms/tests/test_models.py]: try to explain me what this row does:
`self.assertIsInstance(instance, self.model_cls)`

and then same for this one:
`self.assertIsInstance(isinstance(instance, self.model_cls))`

Comment (elya-hp) [reply tms_django/apps/tms/tests/test_models.py]: I don't know it is a kind to check if this testing info are instance I guess 
The isinstance() function in Python is used to check if an object is an instance of a specified class or any of its subclasses

**Pull request merged**